src/test/java/com/gooddata/warehouse/WarehouseServiceAT.java

```diff
@@ -85,6 +94,20 @@ public void shouldListUsers() throws Exception {
         assertThat(users.getNextPage(), is(nullValue()));
     }
 
+    @Test(groups = { "warehouse", "isolated_domain" }, dependsOnMethods = "shouldListUsers")
+    public void shouldAddUserToWarehouse() {
+        account = gd.getAccountService().createAccount(new Account(LOGIN, "nnPvcGXU7f", "FistName", "LastName"), getProperty("domain"));
```

Comment:
I can place account creation in @BeforeClass but it would make whole WarehouseServiceAT as isolated_domain group. Group you asked to exclude from verification.